client/dashboard/domains/name-servers/upsell-nudge.tsx

```diff
@@ -0,0 +1,50 @@
+import { __experimentalText as Text } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { Callout } from '../../components/callout';
+import UpsellCTAButton from '../../components/upsell-cta-button';
+import illustrationUrl from './upsell-nudge-illustration.svg';
+
+interface Props {
+	domainName: string;
+}
+
+export default function UpsaleNudge( { domainName }: Props ) {
+	return (
+		<Callout
+			title={ sprintf(
+				/* translators: %s is the domain name */
+				__( 'This domain is being forwarded to %s' ),
+				domainName
+			) }
+			image={ illustrationUrl }
+			variant="highlight"
+			description={
+				<Text as="p">
+					{ createInterpolateElement(
+						sprintf(
+							// translators: %(domain)s is the domain name
+							__(
+								'Upgrade to a paid plan to make <strong>%s</strong> the primary address that your visitors see when they visit your site.<br /><a>Learn more</a>'
+							),
+							domainName
```

Comment:
This translator note looks wrong. I think it should be this?

```suggestion
							// translators: %(domain)s is the domain name
							__(
								'Upgrade to a paid plan to make <strong>%(domain)s</strong> the primary address that your visitors see when they visit your site.<br /><a>Learn more</a>'
							),
							{ domain: domainName }
```

Reply:
There's no good reason to both have a markup and placeholder like `<strong>%(domain)s</strong>`. Instead, we can just use the markup, `<domain />`. See https://github.com/Automattic/wp-calypso/pull/103700#discussion_r2108720909.

Reply:
Agree! I've simplified it.